components/org.wso2.openbanking.cds.identity/src/main/java/org/wso2/openbanking/cds/identity/metadata/periodical/updater/ExternalMetadataUpdater.java

```diff
@@ -45,6 +48,7 @@ public static ExternalMetadataUpdater getInstance() {
     }
 
     public void updateMetadata() {
+        log.info("Updating data recipient and software product metadata by external call.");
```

Comment:
Debug?

Reply:
How frequent is this?

Reply:
This is only for urgent updates by the regulator. Rarely called.